.gitignore

```diff
@@ -26,3 +26,4 @@ _Build
 _Work
 .vs
 .idea
+User.Directory.Build.props
```

Comment:
Is it OK that this file have no new line end of file? For example User.Directory.Build.props.example is also new but has it. I guess it should be there for consistency across all files.